src/encoded/regulome_indexer.py

```diff
@@ -904,7 +891,7 @@ def metadata_doc(afile, dataset):
             if dataset_type in dataset['@type']:
                 meta_doc['dataset_type'] = dataset_type
                 break
-        if afile.get('submitted_file_name') in SNP_FILES:
+        if dataset_type == 'Reference':
```

Comment:
This seems a little sloppy; there are lots of references that might contain a parsable/indexable file that aren't SNPs.  Not sure I would worry about it now though.

Reply:
When it's here, it should already been filtered above with internal_tag (line 828-835). I only tag SNP reference as RegulomeDB. It's still not a very good solution. Any suggestions?